### PR TITLE
TimofeevaAI_lab1

### PR DESCRIPTION
**Pull request by nastyonai**

### CODE REVIEW

**Comment by nastyonai** [comment]
Help me please, i don't know how to solve the problem - Conditional jump or move depends on uninitialised value(s).

**Comment by nastyonai** [comment]
I understood. I didnt enter numbers in input

File: TimofeevaAI/lab_1/Source.cpp [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+public:
+	Vector();
+	Vector(unsigned size);
+	Vector(unsigned size, const T & val);
+	unsigned Size();
+	void Resize(unsigned newsize);
+	bool Insert(unsigned pos, const T & val);
+	void push_back(const T & val);
+	bool contains(const T & val);
+    T operator[](unsigned index);
```
Comment (Toxa-man) [inline TimofeevaAI/lab_1/Source.cpp]: You are not able to change vector elements thru this operator

File: TimofeevaAI/lab_1/Source.cpp [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+	arr = new T[size];
+	for (unsigned i = 0; i < size; ++i)
+	{
+		arr[i] = T();
+	}
+	for (unsigned i = 0; i < size; ++i)
+	{
+		cout << arr[i] << " ";
+	}
+	cout << endl << endl;
```
Comment (Toxa-man) [inline TimofeevaAI/lab_1/Source.cpp]: There should be no output in the entire vector class

**Review by Toxa-man — COMMENTED** [review]
Code issues

**Pull request merged**